samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -1358,24 +1358,34 @@ mfxStatus CEncodingPipeline::Init(sInputParams *pParams)
 
     if (bVpp)
     {
+		msdk_printf(MSDK_STRING("Note: VPP is enabled.\n"));
```

Comment:
alignment is wrong.

Reply:
alignment is corrrect - tabs are displayed incorrectly